presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -89,6 +89,11 @@
      */
     Optional<PartitioningHandle> getCommonPartitioning(Session session, PartitioningHandle left, PartitioningHandle right);
 
+    /**
+     * Provides partitioning handle for exchange
```

Comment:
`.` at the end.

Reply:
Why not plural (`getPartitioningHandleForExchanges`)?
Or `getExchangePartitioningHandle`?

Reply:
Because we are getting partitioning handles per exchange, as the handle contains the list of types for partitioning columns. So technically there will be one partitioning handle per exchange.